woocommerce/woocommerce-ios
# Retrieving variations and displaying the option for bulk update the regular price

Part of: #6239

### Description
This PR adds in the bulk update view model a results controller to retrieve the product variations that we will update. Also  based on that variations it provides the user facing information to be displayed for the option of updating the regular price, this information is either **none**, **mixed**, or a formatted price if all variations have the same price.

It remains in future PR to adjust the font style of the `ValueOneTableViewCell` to match the designs (we have a sub task for this), and handle the action on the tap event.

### Testing instructions

1. Go to the products tab
2. Select a product with variations
3. Go to the list of variations
4. Tap on the "more" button in the navigation bar
5. Select the "Bulk update"
6. After the view loads, a regular price option should be available. This option should have as a description the following: **None** if all the variations have not "regular price" value, **Mixed** if a least one variation has different value, or a formatted price if all variations have the same price.

### Screenshots

Same Price
![Same Price](https://user-images.githubusercontent.com/96764631/156223651-7d99cbe0-a4cb-430c-bfa1-d410bb682827.png)

Mixed Prices
![Mixed Prices](https://user-images.githubusercontent.com/96764631/156223655-9a195559-d62c-4aea-973e-7e7269c0d217.png)

Prices Not Set
![Prices Not Set](https://user-images.githubusercontent.com/96764631/156223656-df862d66-a648-415d-b3db-cc13aa1362e4.png)


---
